tests/phpunit/integration/Modules/AdsTest.php

```diff
@@ -11,56 +11,181 @@
 namespace Google\Site_Kit\Tests\Modules;
 
 use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
 use Google\Site_Kit\Core\Tags\GTag;
 use Google\Site_Kit\Modules\Ads;
+use Google\Site_Kit\Modules\Ads\Settings;
+use Google\Site_Kit\Tests\Fake_Site_Connection_Trait;
 use Google\Site_Kit\Tests\TestCase;
+use WP_User;
 
 /**
  * @group Modules
  * @group Ads
  */
 class AdsTest extends TestCase {
 
+	private $user_id;
+
+	/**
+	 * Ads object.
+	 *
+	 * @var Ads
+	 */
+	private $ads;
+
+	/**
+	 * @var User_Options
+	 */
+	protected $user_options;
+
+	/**
+	 * @var Options
+	 */
+	protected $options;
+
+	/**
+	 * @var Authentication
+	 */
+	protected $authentication;
+	use Fake_Site_Connection_Trait;
+
+	/**
+	 * Plugin context.
+	 *
+	 * @var Context
+	 */
+	private $context;
+
+	public function set_up() {
+		parent::set_up();
+
+		$administrator = self::factory()->user->create_and_get( array( 'role' => 'administrator' ) );
+		$this->user_id = $administrator->ID;
+		wp_set_current_user( $this->user_id );
+
+		$this->context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+		$this->ads     = new Ads( $this->context );
+
+		self::enable_feature( 'adsPax' );
+	}
+
+	private function force_permissions_for_current_user() {
```

Comment:
Since we won't be including the setup assets [test](https://github.com/google/site-kit-wp/pull/8595#discussion_r1577164256) in this PR, this method won't be needed, I removed it as well